packages/chem/quri_parts/chem/utils/orbital_rotation.py

```diff
@@ -0,0 +1,63 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#      http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Sequence
+
+from quri_parts.circuit import (
+    LinearMappedUnboundParametricQuantumCircuit,
+    Parameter,
+    ParameterOrLinearFunction,
+)
+
+
+def add_orbital_rotation_gate(
```

Comment:
It seems that this can be written simpler as two sequential single excitation circuits.
https://docs.pennylane.ai/en/stable/code/api/pennylane.OrbitalRotation.html
https://docs.pennylane.ai/en/stable/_modules/pennylane/ops/qubit/qchem_ops.html#OrbitalRotation.compute_decomposition